### ISSUE TITLE
15 Keypoints Is All You Need 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#45]
# TL;DR

In this study, they used only key point information in human Pose Tracking, without using RGB or Optical Flow. The key point embedded as token with key point position, type of the point, and time and  then put into the Transformer Encoder to assgn ID. 
![image](https://user-images.githubusercontent.com/24524018/90498714-9bdae200-e183-11ea-8d0b-e284c11580df.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1912.02323

# Submission Dates(yyyy/mm/dd)
2019/12/05
# Authors and institutions
Michael Snower, Asim Kadav, Farley Lai, Hans Peter Graf
- Brown University
- NEC Labs America

# Methods

# Results

# Comments
CVPR2020
